apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimResponseSearch.json

```diff
@@ -1,14 +1,14 @@
 {
   "resourceType": "Bundle",
-  "id": "1856fb88-119e-46b6-a775-ab78ae0f61e9",
+  "id": "0561fc36-9def-493f-9be0-9354e139b81b",
   "meta": {
-    "lastUpdated": "1970-01-01T00:00:01.000+00:00"
+    "lastUpdated": "2021-12-16T18:44:32.354+00:00"
```

Comment:
Why are these dates changing?  Can't the test data just use the old date?

Reply:
Yea, sometimes I don't catch all the deltas, the `lastUpdated` and a few other values are ignored for the diff, so the test passes still, but every time the test is run, the values change.  Side effect of a lazy copy over and not reverting the ignored path deltas